def raise_exception():
    raise TypeError("This is a custom type error.")